ldk/go/filesystemServer.go

```diff
@@ -2,18 +2,24 @@ package ldk
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"os"
 
 	"github.com/golang/protobuf/ptypes"
 	"github.com/open-olive/loop-development-kit/ldk/go/proto"
+	"google.golang.org/protobuf/types/known/emptypb"
 )
 
 // FilesystemServer is used by the controller plugin host to receive plugin initiated communication
 type FilesystemServer struct {
 	Impl FilesystemService
 }
 
-// list the contents of a directory
+// ErrNoFile File is not initiated
+var ErrNoFile = errors.New("no file")
```

Comment:
Can you please make this error more verbose. Something like:
`received invalid request to act on a file before a file has been opened or created`